### ISSUE TITLE
Capes clip out when used with Elytra Trinket

### ISSUE THREAD

**Issue by sorat0mo** [tr7zw/WaveyCapes#9]
What happened:
The cape occasionally clips through when an elytra is equipped in the elytra slot alongside a chestplate.

Expected:
The cape disappears or the wavey physics is applied to elytra instead.

**Comment by tr7zw** [comment]
The issue is that the mod currently doesn't know about "an elytra slot". I guess this is talking about Trinkets?

**Comment by sorat0mo** [comment]
Yes. [Elytra Trinket](https://www.curseforge.com/minecraft/mc-mods/elytra-trinket-fabric). I saw it is listed as **Tested With**. The mod sill function as is, but it sticks out when both an armor piece is equipped and elytra is equipped:

![2022-01-12_20 58 15](https://user-images.githubusercontent.com/11853019/149144800-469b9f43-7de7-4f44-ba42-f3adc28ad5bd.png)

If the elytra is not equipped in chestplate slot(vanilla slot), this problem does not occur.

**Comment by tr7zw** [comment]
I think it was tested with trinkets in regards to using banners as capes, not having an extra elytra slot. Will fix that somewhere in the future.

**Comment by TatleCat** [comment]
> 我认为它已经用小饰品进行了测试，将旗帜用作斗篷，没有额外的鞘翅槽。将在将来的某个地方解决这个问题。

This issue is still occurring with these two mods in the latest Fabric-1.20.1

**Comment by tr7zw** [comment]
Any Trinkets mod is still not supported.

**Comment by ArduFish123** [comment]
Can confirm in 1.20.6
